CMakeLists.txt

```diff
@@ -42,9 +42,14 @@ FetchContent_Populate(hclust)
 add_library ("fastcluster" STATIC ${hclust_SOURCE_DIR}/fastcluster.cpp)
 target_include_directories ("fastcluster" PUBLIC ${hclust_SOURCE_DIR})
 
+# Dependency: BAMIntervalTree.
 add_library ("bamit" INTERFACE)
 target_include_directories ("bamit" INTERFACE lib/BAMIntervalTree/include/)
 
+# Dependency: b.i.o.
+add_library ("bio" INTERFACE)
+target_include_directories ("bio" INTERFACE lib/b.i.o./include/)
```

Comment:
```suggestion
find_package (bio QUIET REQUIRED HINTS lib/bio/build_system)
```

Reply:
Thsi ends up in an error aswell:
```bash
[ 82%] Building CXX object src/CMakeFiles/iGenVar_lib.dir/variant_detection/variant_output.cpp.o
/Users/lydia/Repos/iGenVar/src/variant_detection/variant_output.cpp:8:10: fatal error: bio/var_io/writer.hpp: No such file or directory
    8 | #include <bio/var_io/writer.hpp>        // for bio::var_io::header
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```
Do I have to do something else?

Reply:
Hm, this is strange, and it should definitely work. Can you try doing the find_package before SeqAn's find_package? In fact, you shouldn't need SeqAn's find_package, because bio already finds SeqAn3.